docs/SOP-add_OBA_terms_for_GWAS.md

```diff
@@ -0,0 +1,103 @@
+How to add new GWAS requested trait terms to OBA and EFO?
+=========================================================
+
+
+## 1. Check if an appropriate OBA trait pattern already exists
+Look into [bio-attribute-ontology/src/patterns/dosdp-patterns/](https://github.com/obophenotype/bio-attribute-ontology/tree/master/src/patterns/dosdp-patterns) and check if the GWAS trait term would fit into any of the existing patterns.
+- [ ] If yes, skip to the next step.
+- [ ] If none of the existing OBA trait patterns look appropriate, then create a new pattern.
+In some cases, the requested GWAS term may not fit the scope of OBA. In that case, an new EFO term can be created without an equivalent OBA trait term.
+
+## 2. Create new OBA term(s)
+Fill in the appropriate [DOS-DP template data table](https://oboacademy.github.io/obook/tutorial/dosdp-odk/) to add any new terms to OBA in [bio-attribute-ontology/src/patterns/data/default/](https://github.com/obophenotype/bio-attribute-ontology/tree/master/src/patterns/data/default).
+For example, for a trait involving the 'age at which disease manifestations first appear', fill in the table [`disease_onset.tsv`.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/disease_onset.tsv).
+Create a unique OBA identifier by using the next available ID from your [assigned range.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/oba-idranges.owl)
+
+Keep track of the IDs from your range that you have already assigned.
+
+Also fill in the appropriate terms as specified in the actual DOS-DP `yaml` template file.
+For example, in the case of the [`disease_onset.tsv`.](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/disease_onset.tsv) table, you must use MONDO [disease or disorder](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0000001) terms in the `disease` column.
+
+## 3. OBA release
+
+The newly created trait terms can be imported into EFO from a publicly released version of OBA.   
+To run the OBA release pipeline, follow the steps below.
+
+* Make sure you have the latest ODK installed by running docker
+
+    docker pull obolibrary/odkfull
+
+* Merge as many open PRs as possible
+* Start with a fresh copy of the `master` branch
+
+    git pull
+
+* Checkout a new branch
+
+    git checkout -b release-202X-XX-XX
+
+* In a terminal window, start the [release pipeline:](http://pato-ontology.github.io/pato/odk-workflows/ReleaseWorkflow/)
```

Comment:
Wouldnt it be better to create our own?

Reply:
Added a new document `OBA_ReleaseWorkflow.md`.